EventEnabledInsurance/prereqs.sh

```diff
@@ -132,6 +132,35 @@ fi
 
 echo -e "\n----------------------------------------------------------------------------------------------------------------------------------------------------------\n"
 
+REPLICATION_USER=$(echo ${namespace}_sor_reeplication_${SUFFIX} | sed 's/-/_/g')
```

Comment:
I think that's a spelling mistake there?
```suggestion
REPLICATION_USER=$(echo ${namespace}_sor_replication_${SUFFIX} | sed 's/-/_/g')
```